tests/integration/common_tasks/common_tasks/test_tasks/fib.py

```diff
@@ -0,0 +1,19 @@
+from girder_worker.utils import girder_job
+from girder_worker.app import app
+
+
+@girder_job(title='Fibonacci Job')
+@app.task
+def fibonacci(n, **kwargs):
+    if n == 1 or n == 2:
+        return 1
+    return fibonacci(n-1) + fibonacci(n-2)
+
+
+@app.task(bind=True)
+def fib_seq(task, n, **kwargs):
+    if n < 0:
+        raise Exception('Must pass in positive integer!')
+
+    for _n in range(1, n+1):
+        print '%s: %s' % (_n, fibonacci(_n))
```

Comment:
We should make sure these tests run under Python 3 before we merge, to avoid adding more work to #119.